components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -239,13 +286,41 @@ def load_credentials_from_file(self, credentials_file_path: pathlib.Path):
                 f"Credentials file '{credentials_file_path}' does not contain key '{ex}'."
             )
 
+        try:
+            self.root_password = get_config_value(config, f"{DB_COMPONENT_NAME}.root_password")
+        except KeyError:
+            pass
+
+        try:
+            self.privileged_username = get_config_value(
+                config, f"{DB_COMPONENT_NAME}.privileged_user"
+            )
+            self.privileged_password = get_config_value(
+                config, f"{DB_COMPONENT_NAME}.privileged_password"
+            )
+        except KeyError:
+            pass
+
     def load_credentials_from_env(self):
         """
-        :raise ValueError: if any expected environment variable is not set.
+        Loads database credentials from environment variables.
+
+        :raise ValueError: If any expected environment variable is not set.
         """
         self.username = _get_env_var(CLP_DB_USER_ENV_VAR_NAME)
         self.password = _get_env_var(CLP_DB_PASS_ENV_VAR_NAME)
 
+        try:
+            self.root_password = _get_env_var(CLP_DB_ROOT_PASS_ENV_VAR_NAME)
+        except ValueError:
+            pass
+
+        try:
+            self.privileged_username = _get_env_var(CLP_DB_PRIVILEGED_USER_ENV_VAR_NAME)
+            self.privileged_password = _get_env_var(CLP_DB_PRIVILEGED_PASS_ENV_VAR_NAME)
+        except ValueError:
+            pass
```

Comment:
instead of this, can we write:
```
    def load_credentials_from_env(self):
        """
        Loads database credentials for the CLP user from environment variables.
        :raise ValueError: If any expected environment variable is not set.
        """
        self.username = _get_env_var(CLP_DB_USER_ENV_VAR_NAME)
        self.password = _get_env_var(CLP_DB_PASS_ENV_VAR_NAME)

    def load_admin_credentials_from_env(self):
        """
        Loads database credentials for the CLP admin user from environment variables.

        :raise ValueError: If any expected environment variable is not set.
        """
        self.admin_username = _get_env_var(CLP_DB_ADMIN_USER_ENV_VAR_NAME)
        self.admin_password = _get_env_var(CLP_DB_ADMIN_PASS_ENV_VAR_NAME)
```